synapseclient/api/file_services.py

```diff
@@ -48,12 +49,30 @@ async def post_file_multipart(
     )
 
 
+@dataclass
+class AddPartResponse:
+    """Result of a part add.
+
+    Attributes:
+        upload_id: The unique identifier of a multi-part request.
+        part_number: The part number of the add.
+        add_part_state: The state of this add.
+        error_message: If the added failed, this will contain the error message of the
+            cause. Will be None when the add is successful.
+    """
+
+    upload_id: str
+    part_number: int
+    add_part_state: str
```

Comment:
Is this an enum?

Reply:
The REST API has this as an Enum. I didn't do that conversion in this case.